package.json

```diff
@@ -58,6 +58,7 @@
     "babel-preset-stage-1": "6.24.1",
     "css-loader": "0.28.11",
     "electron": "2.0.0",
+    "electron-debug": "2.0.0",
```

Comment:
Since we are doing this, lets install `devtron` and `electron-react-devtools` too.

`npm install --save-dev devtron electron-react-devtools`

Reply:
I'll look into those, thanks!